docs/reference/announcements-release-notes/850/850-announcements.md

```diff
@@ -1,16 +1,17 @@
 ---
-id: announcements-850
+id: 850-announcements
 title: "8.5 - 8.3 Announcements"
 description: "Important announcements including deprecation & removal notices for the Camunda 8.5, 8.4, 8.3, and 8.2 releases."
+toc_max_heading_level: 2
```

Comment:
@pepopowitz @akeller

I realized I have been furiously battling PRs because Markdown All in One has been auto-formatting my TOCs when I save literally any page, which has been requiring me to manually go into GH and update the page with the TOC I want.

Now that I have disabled the extension, is this an appropriate way for me to manage the TOC, or is there a preferred alternative?

CC @mesellings for raising this change 👍

Reply:
Sharing a screenshot of how this plays out in the preview site: 

<img width="1422" alt="image" src="https://github.com/user-attachments/assets/f03694e2-a663-4b62-8f1b-d918fcd970e5" />

as compared to a table of contents like this 😵  :

<img width="324" alt="image" src="https://github.com/user-attachments/assets/03f5aa64-e942-4547-9054-d99e7cb3f874" />

I like this a lot!!!!